docs/hugo/content/guide/breaking-changes/breaking-changes-v2.9.0.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "v2.8.0 Breaking Changes"
+linkTitle: "v2.8.0"
+weight: -25  # This should be 5 lower than the previous breaking change document
+---
+
+# DelegatedManagedIdentityResourceId is now a secret
+
+The RoleAssignment property `.spec.DelegatedManagedIdentityResourceId` has been changed from a string to a SecretReference and renamed to `.spec.DelegatedManagedIdentityResourceReference`.
```

Comment:
```suggestion
The RoleAssignment property `.spec.delegatedManagedIdentityResourceId` has been changed from a string to a SecretReference and renamed to `.spec.delegatedManagedIdentityResourceReference`.
```

I think is the correct casing, since we're referring to it by JSON path?